### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

Note: I'm pretty sure I've broken some "public APIs". I'm going to wait for input wrt how to address such breaks.

### CODE REVIEW

File: community/codegen/src/main/java/org/neo4j/codegen/bytecode/ByteCodeVerifier.java [hunk 1]
```diff
@@ -349,7 +349,7 @@ protected boolean isAssignableFrom( Type target, Type value )
         protected boolean isSubTypeOf( BasicValue value, BasicValue expected )
         {
             return super.isSubTypeOf( value, expected ) || check
-                    .invokableInterface( expected.getType(), value.getType() );
+                    .invocableInterface( expected.getType(), value.getType() );
```
Comment (martinfurmanski) [inline community/codegen/src/main/java/org/neo4j/codegen/bytecode/ByteCodeVerifier.java:352]: What's wrong with invokable?

File: community/consistency-check/src/test/java/org/neo4j/consistency/checking/SchemaRecordCheckTest.java [hunk 2]
```diff
@@ -50,11 +50,11 @@
 
     public SchemaRecordCheckTest()
     {
-        super( new SchemaRecordCheck( configureSchemaStore(), configureIndexAcessors() ),
+        super( new SchemaRecordCheck( configureSchemaStore(), configureIndexAssessors() ),
```
Comment (martinfurmanski) [inline community/consistency-check/src/test/java/org/neo4j/consistency/checking/SchemaRecordCheckTest.java]: This is not an assessor, but an accessor.

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/ClosingIteratorTest.scala [hunk 3]
```diff
@@ -37,8 +37,8 @@ class ClosingIteratorTest extends CypherFunSuite {
 
   test("should not close prematurely") {
     //Given
-    val wrapee   = Iterator(Map("k" -> intValue(42)))
-    val iterator = new ClosingIterator(wrapee, taskCloser, exceptionDecorator)
+    val wrappee   = Iterator(Map("k" -> intValue(42)))
```
Comment (martinfurmanski) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/ClosingIteratorTest.scala]: Please fix alignment. Is this even a real word?

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/ClosingIteratorTest.scala [hunk 4]
```diff
@@ -49,8 +49,8 @@ class ClosingIteratorTest extends CypherFunSuite {
 
   test("should cleanup even for empty iterator") {
     //Given
-    val wrapee   = Iterator.empty
-    val iterator = new ClosingIterator(wrapee, taskCloser, exceptionDecorator)
+    val wrappee   = Iterator.empty
```
Comment (martinfurmanski) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/ClosingIteratorTest.scala]: Alignment.

File: community/cypher/cypher-planner-3.4/src/main/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/logical/StatisticsBackedCardinalityModel.scala [hunk 5]
```diff
@@ -34,10 +34,10 @@ class StatisticsBackedCardinalityModel(queryGraphCardinalityModel: QueryGraphCar
   def apply(query: PlannerQuery, input0: QueryGraphSolverInput, semanticTable: SemanticTable): Cardinality = {
     val output = query.fold(input0) {
       case (input, RegularPlannerQuery(graph, horizon, _)) =>
-        val QueryGraphSolverInput(newLabels, graphCardinality, lazyness) = calculateCardinalityForQueryGraph(graph, input, semanticTable)
+        val QueryGraphSolverInput(newLabels, graphCardinality, laziness) = calculateCardinalityForQueryGraph(graph, input, semanticTable)
```
Comment (martinfurmanski) [inline community/cypher/cypher-planner-3.4/src/main/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/logical/StatisticsBackedCardinalityModel.scala]: What's wrong with lazyness?

File: community/kernel/src/main/java/org/neo4j/kernel/impl/store/PropertyStore.java [hunk 6]
```diff
@@ -128,7 +128,7 @@
  * DATE:      [    ,    ] [    ,    ] [    ,    ] [    ,   0] [ 01 ,type][K][K][K] epochDay in next long block
  * LOCALTIME: [xxxx,xxxx] [xxxx,xxxx] [xxxx,xxxx] [xxxx,xxx1] [ 02 ,type][K][K][K] nanoOfDay
  * LOCALTIME: [    ,    ] [    ,    ] [    ,    ] [    ,   0] [ 02 ,type][K][K][K] nanoOfDay in next long block
- * LOCALDTIME:[xxxx,xxxx] [xxxx,xxxx] [xxxx,xxxx] [xxxx,xxxx] [ 03 ,type][K][K][K] nanoOfSecond
+ * LOCALTIME:[xxxx,xxxx] [xxxx,xxxx] [xxxx,xxxx] [xxxx,xxxx] [ 03 ,type][K][K][K] nanoOfSecond
```
Comment (Lojjs) [inline community/kernel/src/main/java/org/neo4j/kernel/impl/store/PropertyStore.java]: I think this is not correct, the`D` here is short for `DATE` as in `LocalDateTime`

File: community/cypher/cypher-planner-3.4/src/main/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/logical/StatisticsBackedCardinalityModel.scala (hunk 5 above)
Comment (sherfert) [reply community/cypher/cypher-planner-3.4/src/main/scala/org/neo4j/cypher/internal/compiler/v3_4/planner/logical/StatisticsBackedCardinalityModel.scala]: "Laziness" is the correct spelling

File: community/codegen/src/main/java/org/neo4j/codegen/bytecode/ByteCodeVerifier.java (hunk 1 above)
Comment (sherfert) [reply community/codegen/src/main/java/org/neo4j/codegen/bytecode/ByteCodeVerifier.java:352]: It's debatable, but according to this "invocable" should be preferred: https://en.wiktionary.org/wiki/Talk:invokable
Anyways, both seem to be correct, though.

File: community/consistency-check/src/test/java/org/neo4j/consistency/checking/SchemaRecordCheckTest.java (hunk 2 above)
Comment (jsoref) [reply community/consistency-check/src/test/java/org/neo4j/consistency/checking/SchemaRecordCheckTest.java]: Oops, will fix, thanks.

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/ClosingIteratorTest.scala (hunk 3 above)
Comment (jsoref) [reply community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/ClosingIteratorTest.scala]: Will do. It isn't a real word afaict, but per language rules wrappee is the proper spelling based on wrapper.

File: community/kernel/src/main/java/org/neo4j/kernel/impl/store/PropertyStore.java (hunk 6 above)
Comment (jsoref) [reply community/kernel/src/main/java/org/neo4j/kernel/impl/store/PropertyStore.java]: Oops, will drop, thanks.

**Comment by jsoref** [comment]
Rebased

**Comment by sherfert** [comment]
@jsoref Currently failing with:
```
[Step 2/5] [INFO] Missing header in: /mnt/teamcity/work/7c591132dbf26b78/community/kernel/src/test/java/org/neo4j/graphdb/schema/UpdateDeletedIndexIT.java
[07:09:58][Step 2/5] Failed to execute goal org.neo4j.build.plugins:license-maven-plugin:3:check (default-cli) on project neo4j-kernel: Some files do not have the expected license header
```

**Comment by jsoref** [comment]
@sherfert : that looks like something that would be fixed by a rebase.

 This is a fast moving project. I won't have time to touch it until Monday...

**Comment by sherfert** [comment]
@jsoref Yeah, PRs that touch lots of files are very hard to get in, I agree.

**Comment by martinfurmanski** [comment]
@jsoref Please squash the PR as well.

**Comment by jsoref** [comment]
Squashing makes rebasing changes like this very very very hard -- (I'll have to privately maintain the individual commits for the average case where it needs to be rebased again).

@sherfert / @martinfurmanski : could you explain why the builds failed this time around?

I'm happy to do one or maybe two squashes (although you should probably suggest the commit message -- normally I squash in the form:

```
Spelling fixes

* acceptable
...
* frontage
...
...
* zoom
```

**Comment by martinfurmanski** [comment]
@jsoref You can probably ignore the mention of which specific words you are fixing in the commit message. It will be obvious from the commit itself. The important thing is to avoid 186 commits in the history. It is sufficient with one saying "Fixing spelling errors".

**Comment by jsoref** [comment]
Ok. I can use that message. I still need to know why the build is failing now.

**Comment by sherfert** [comment]
```
[Step 5/6] [ERROR] COMPILATION ERROR : 
[14:37:42][Step 5/6] [INFO] -------------------------------------------------------------
[14:37:42][Step 5/6] [ERROR] /mnt/teamcity/work/7c591132dbf26b78/community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/centrality/ParallelCentralityCalculation.java:[49,20] cannot find symbol
[14:37:42][Step 5/6]   symbol:   class ShortestPathBasedCentrality
[14:37:42][Step 5/6]   location: class org.neo4j.graphalgo.impl.centrality.ParallelCentralityCalculation<ShortestPathCostType>
[14:37:42][Step 5/6] [ERROR] /mnt/teamcity/work/7c591132dbf26b78/community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/centrality/ParallelCentralityCalculation.java:[75,9] cannot find symbol
[14:37:42][Step 5/6]   symbol:   class ShortestPathBasedCentrality
[14:37:42][Step 5/6]   location: class org.neo4j.graphalgo.impl.centrality.ParallelCentralityCalculation<ShortestPathCostType>
```

This does not look like you introduced that, but rather were unlucky to rebase onto a broken state - I guess.

@martinfurmanski We can alternatively choose to squash while merging, can't we?

**Comment by martinfurmanski** [comment]
@sherfert Yes that works as well!

**Comment by jsoref** [comment]
It looks like the file I was renaming was deleted and my rebase didn't properly handle that...

**Comment by jsoref** [comment]
@sherfert: I've rebased, I don't see the bot anymore...

**Comment by sherfert** [comment]
@jsoref We had some CI hickups. You'll have to rebase again, given the conflicting files.

**Comment by jsoref** [comment]
@sherfert: rebased, but it's complaining again (and I can't see why).

**Comment by systay** [comment]
Hi @jsoref 

We have changed the license headers for source code files since you started this PR. (Sorry it's taken so long!)

If you execute `mvn license:format`, a bunch of files will have their headers updated. That should clear out the build problem.

**Comment by jsoref** [comment]
@sherfert / @martinfurmanski / @systay: ok, hopefully that covers things...

**Comment by sherfert** [comment]
Current failure:

```
[05:05:10][Step 5/6] [ERROR] /mnt/teamcity/work/7c591132dbf26b78/community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_5/runtime/ClosingIteratorTest.scala:29: error: ClosingIteratorTest is already defined as class ClosingIteratorTest
[05:05:10][Step 5/6] [ERROR] class ClosingIteratorTest extends CypherFunSuite {
```

Looking at your diff at seems like you added this file. What I think happened is that is was renamed since this PR was opened. Please have a look if that file exists twice now.

**Pull request closed without merging**